### PR TITLE
Add German translation for Tobira-related Admin UI strings

### PR DESCRIPTION
**Pull request by LukasKalbertodt**

### CODE REVIEW

File: modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json [hunk 1]
```diff
@@ -245,7 +245,14 @@
       "ADOPTER_PROFILE_UPDATED": "Profil-Informationen erfolgreich aktualisiert.",
       "ADOPTER_PROFILE_NOT_UPDATED": "Hoppla! Profil-Informationen konnten nicht aktualisiert werden. Bitte versuchen Sie es erneut!",
       "ADOPTER_PROFILE_DELETED": "Profil-Informationen erfolgreich gelöscht.",
-      "ADOPTER_PROFILE_NOT_DELETED": "Hoppla! Profil-Informationen konnten nicht gelöscht werden. Bitte versuchen Sie es erneut!"
+      "ADOPTER_PROFILE_NOT_DELETED": "Hoppla! Profil-Informationen konnten nicht gelöscht werden. Bitte versuchen Sie es erneut!",
+      "TOBIRA_NO_TITLE": "Seite benötigt einen Titel",
+      "TOBIRA_NO_PATH_SEGMENT": "Seite benötigt ein Pfadsegment",
+      "TOBIRA_NOT_MOUNTED": "Serie konnte nicht in Tobira eingehangen werden",
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json:251]: Isn't it "eingehängt"? 🤔

File: modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json [hunk 2]
```diff
@@ -245,7 +245,14 @@
...
+      "TOBIRA_PATH_SEGMENT_INVALID": "Pfadsegment ist ungültig. Ein gültiges Pfadsegment ist mindestens zwei Zeichen lang, enthält keine Leerzeichen, Kontrollzeichen, und keine der Zeichen '<>\"[\\]^`{|}#%\/?]', und fängt mit keinem der Zeichen '-+~@_!$&;:.,=*'()' an.",
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json:252]:
```suggestion
      "TOBIRA_PATH_SEGMENT_INVALID": "Pfadsegment ist ungültig. Ein gültiges Pfadsegment ist mindestens zwei Zeichen lang, enthält keine Leerzeichen, Kontrollzeichen, keins der Zeichen '<>\"[\\]^`{|}#%\/?]' und fängt mit keinem der Zeichen '-+~@_!$&;:.,=*'()' an.",
```

File: modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json [hunk 3]
```diff
@@ -245,7 +245,14 @@
...
+      "TOBIRA_PATH_SEGMENT_UNIQUE": "Das gleiche Pfadsegment darf nicht mehrmals für Seiten mit der gleichen Überseite verwendet werden.",
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json:253]: Some validation messages have punctuation, others don't. Should we unify that? 🤔

File: modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json [hunk 4]
```diff
@@ -245,7 +245,14 @@
...
+      "TOBIRA_PAGE_NOT_FOUND": "Die angefragte Seite existiert nicht (mehr). Versuchen Sie wieder vom Anfang zu beginnen.",
+      "TOBIRA_SERVER_ERROR":" Fehler bei der Kommunikation mit Tobira. Kontaktieren Sie Ihren Systemadministrator oder versuchen Sie es später neu."
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json:255]:
```suggestion
      "TOBIRA_SERVER_ERROR":" Fehler bei der Kommunikation mit Tobira. Kontaktieren Sie Ihren Systemadministrator oder versuchen Sie es später erneut."
```

?

File: modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json [hunk 5]
```diff
@@ -1186,6 +1208,12 @@
                "LICENSE": "Lizenz",
                "DESCRIPTION": "Beschreibung"
             },
+            "TOBIRA": {
+              "HOMEPAGE": "(Startseite)",
+              "DIRECT_LINK": "Direktlink zur Serie",
+              "NOT_MOUNTED": "Diese Serie zurzeit auf keiner Seite angezeigt",
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json:1214]: Is this missing a verb?

**Comment by LukasKalbertodt** [comment]
I agree with all comments. Wanna open another PR fixing all that? :D

**Pull request merged**